app/src/main/java/eu/kanade/tachiyomi/ui/library/DisplayBottomSheet.kt

```diff
@@ -72,6 +73,32 @@ class DisplayBottomSheet(private val controller: LibraryController) : BottomShee
                     settings_scroll_view.paddingTop + settings_scroll_view.paddingBottom
             close_button.visibleIf(isScrollable)
         }
+        updateRadioGroupScrollBar()
+        addCheckOnChangeLisctenerToScrollBar()
+    }
+
+    private fun addCheckOnChangeLisctenerToScrollBar() {
+        Show_hide_auto_scrollbar.setOnCheckedChangeListener(RadioGroup.OnCheckedChangeListener { radioGroup, i ->
+            if (i == R.id.radioButton_never) {
+                preferences.showHideScrollBar().set(0)
+            } else if (i == R.id.radioButton2_auto) {
+                preferences.showHideScrollBar().set(1)
+            } else if (i == R.id.radioButton3_always) {
+                preferences.showHideScrollBar().set(2)
+            }
+            controller.updateScrollBar()
+        })
+    }
+
+    private fun updateRadioGroupScrollBar() {
+        val prefScrollBar = preferences.showHideScrollBar().getOrDefault().toInt()
+        if (prefScrollBar == 0) {
+            radioButton_never.isChecked = true
+        } else if (prefScrollBar == 1) {
+            radioButton2_auto.isChecked = true
+        } else if (prefScrollBar == 2) {
+            radioButton3_always.isChecked = true
+        }
```

Comment:
you should be using the bindToPreference like the rest of the preferences vs manually doing it here.  see the display_group as an example since it is a radio button also